conventions/src/main/kotlin/otel.scala-conventions.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
 }
 
 dependencies {
+  compileOnly("org.scala-lang:scala-library:2.11.12")
```

Comment:
IntelliJ was having trouble syncing the project without this. It seemed harmless so hopefully it's ok.